RippleCarryAdder/ReferenceImplementation.qs

```diff
@@ -73,28 +73,24 @@ namespace Quantum.Kata.RippleCarryAdder {
     // Task 1.7. N-bit adder
     operation ArbitraryAdder_Reference (a : Qubit[], b : Qubit[], sum : Qubit[], carry : Qubit) : Unit is Adj {
         let N = Length(a);
-        if (N == 1) {
-            LowestBitSum_Reference(a[0], b[0], sum[0]);
-            LowestBitCarry_Reference(a[0], b[0], carry);
+        if N == 1 {
+            OneBitAdder(a[0], b[0], sum[0], carry);
```

Comment:
You have to use `_Reference` versions of all tasks you use in the reference solutions, since the regular versions are empty in the project (you can see that the tests are failing on this kata in the CI builds)

Reply:
Thanks!